.github/workflows/code-lint.yml

```diff
@@ -7,6 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Setup Node.js
```

Comment:
Without this Node.js 18 is used, which leads to https://github.com/energywebfoundation/ssi-hub/actions/runs/4222857695/jobs/7331874584